docs/SCM-BestPractices/README.md

```diff
@@ -10,21 +10,16 @@ Collaborative source code management platforms (such as GitHub and GitLab) play
 
 This guide has been written for the:
 
-* **Maintainer** who wants to improve the security posture for one or more GitHub repositories or GitLab projects they support.  
+* **Maintainer** who wants to improve the security posture for one or more GitHub repositories or GitLab projects they support.
+* **Owner** who wants to improve the security posture for their GitHub organization or GitLab group they manage. 
 * **Open Source Program Office (OSPO)** (or a team that plays a similar role) who is typically responsible for multiple GitHub organizations or GitLab groups.
 * **Operations** team tasked with applying policies as part of their work managing assets on these platforms.
+* **GitHub/GitLab server administrator** who wants to improve the security posture for their SCM server. 
 
 ## Tooling
 
 Below is a non-exhaustive list of possible tools that can be used to assist in review source code repositories.
 
-### Allstar - <https://github.com/ossf/allstar>
```

Comment:
we should keep this section alphabetical

Reply:
Legitify should be the first suggested tool, as it's the only one that covers all of the guide's policies